src/model/consent/consent.ts

```diff
@@ -42,45 +42,62 @@
  * of 'createdAt' field may need to be tested in the future.
  */
 
-import { NotFoundError, RevokedConsentModificationError } from '../errors'
-import Knex from 'knex'
+import { NotFoundError } from '../errors'
+import Knex from 'knex';
 
 /*
  * Interface for Consent resource type
  */
 export interface Consent {
   id: string;
-  initiatorId?: string;
-  participantId?: string;
+  // participant DFSP that requested a Consent resource be made
+  participantId: string;
+  // status of the consent
   status: string;
```

Comment:
Should be of type `ConsentStatus`

Reply:
Hmm `auth-service` used ACTIVE/REVOKED but since we are re-doing this might as well make it the same as the api currently i.e VERIFIED/REVOKED

Reply:
Yes please.